src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,172 @@
+package duke;
+
+import java.time.LocalDate;
+
+import duke.command.AddCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkDoneCommand;
+import duke.command.MarkUndoneCommand;
+import duke.command.OnDateCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
```

Comment:
I like that you explicitly imported each class instead of a wildcard import here.